### DISCUSSION TITLE
Example to configure color for Select

### DISCUSSION THREAD

**Discussion by willliamchan** [Clooos/Bubble-Card#674] [Questions about config, custom styles and templates]
Hey Clooos

Just a quick question, can we add some guidance re setting up colors for bubble select?

Thanks!!

**Comment by Clooos** [comment]
Hi, I haven't tried yet and it's possible that it is not possible at the moment. 

I will have to check that!

**Comment by MrBearPresident** [comment]
Have you solved this?
Can this be closed?
